### PR TITLE
feat(route): snow-peak new arrivals

### PR DESCRIPTION
**Pull request by IvanWng97**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/snowpeak/us/new-arrivals
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/en/shopping.md [hunk 1]
```diff
@@ -174,6 +174,12 @@ Language
 
 <RouteEn author="nczitzk" example="/shopback/shopee-mart" path="/shopback/:store" :paramsDesc="['Store, can be found in URL']"/>
 
+## Snow Peak
+
+### New Arrivals(USA)
+
+<RouteEn author="NavePnow" example="/snow-peak/new-arrivals" path="/snow-peak/new-arrivals"/>
```
Comment (TonyRL) [inline docs/en/shopping.md:181]: Please change path to `/snowpeak/new-arrivals`

File: lib/v2/snow-peak/new-arrivals.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        url,
+    });
+    const data = response.data;
+
+    const $ = cheerio.load(data);
+    const list = $('.element.product-tile')
+        .map(function () {
+            const data = {};
+            const details = $(this).find('.product-data').eq(0).attr('data-product');
+            const product = JSON.parse(details);
```
Comment (TonyRL) [inline lib/v2/snow-peak/new-arrivals.js:19]:
```suggestion
            const product = $(this).find('.product-data').data('product');
```
You can use [.data()](https://cheerio.js.org/docs/api/classes/Cheerio#data)

File: lib/v2/snow-peak/new-arrivals.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            data.title = product.title;
+            const imgUrl = product.featured_image;
```
Comment (TonyRL) [inline lib/v2/snow-peak/new-arrivals.js:21]: I would prefer to show all the images

File: lib/v2/snow-peak/new-arrivals.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            const img = art(path.join(__dirname, 'templates/new-arrivals.art'), {
+                imgUrl,
+            });
+            data.description = img + product.description;
+            data.link = `${host}/products/${product.handle}`;
+
```
Comment (TonyRL) [inline lib/v2/snow-peak/new-arrivals.js:27]: Please add `pubDate` and `category` from `published_at` and `tags`

File: lib/v2/snow-peak/new-arrivals.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        .map(function () {
+            const data = {};
+            const details = $(this).find('.product-data').eq(0).attr('data-product');
+            const product = JSON.parse(details);
+            data.title = product.title;
+            const imgUrl = product.featured_image;
+            const img = art(path.join(__dirname, 'templates/new-arrivals.art'), {
+                imgUrl,
+            });
+            data.description = img + product.description;
```
Comment (TonyRL) [inline lib/v2/snow-peak/new-arrivals.js:25]: Maybe it's better to include `variants` in the description

File: lib/v2/snow-peak/new-arrivals.js (hunk 3 above)
Comment (NavePnow) [reply lib/v2/snow-peak/new-arrivals.js:21]: some products have up to ten images, so maybe it is not a good idea.

**Review by TonyRL — COMMENTED** [review]
`pubDate` is still missing from feed output

**Comment by NavePnow** [comment]
> 

fix done! thanks!

**Pull request merged**